libnetwork/drivers/overlay/ov_network.go

```diff
@@ -216,6 +216,19 @@ func (d *driver) DeleteNetwork(nid string) error {
 	doPeerFlush = true
 	delete(d.networks, nid)
 
+	if n.secure {
+		for _, s := range n.subnets {
+			if err := programMangle(s.vni, false); err != nil {
+				logrus.WithError(err).Warnf(
+					"Failed to clean up iptables rules set for the subnet %s of network %s", s.subnetIP, n.id)
+			}
+			if err := programInput(s.vni, false); err != nil {
+				logrus.WithError(err).Warnf(
+					"Failed to clean up iptables rules set for the subnet %s of network %s", s.subnetIP, n.id)
+			}
```

Comment:
Guess same could be considered here

Both warnings have the same message; is the `err` itself enough to find which one of these caused the error? (or should the `Warn()` have a slightly different output?